Controller/ProcessController.php

```diff
@@ -104,23 +104,22 @@ public function index(
         /* @var \Kontrolgruppen\CoreBundle\Entity\User $user */
         $user = $this->getUser();
 
-        $paginatorOptions = [];
+        $paginatorOptions = [
+            'defaultSortFieldName' => 'e.caseNumber',
+            'defaultSortDirection' => 'desc',
+        ];
 
         // Get sort and direction from user settings.
-        if (!$request->query->has('sort') || !$request->query->has('direction')) {
+        if (!$request->query->has('sort') && !$request->query->has('direction')) {
```

Comment:
No error is generated but this allows null values if only one of the query params is set resulting in strange data. If the direction key is set but not the sort key, it will store null as sort in the db. Maybe we can ignore this.